lms/djangoapps/courseware/views/views.py

```diff
@@ -47,6 +47,7 @@
 from enrollment.api import add_enrollment
 from course_modes.models import CourseMode
 from lms.djangoapps.grades import course_grades, progress as grades_progress
```

Comment:
`grades_progress` can be removed, `course_grades` is used here as well, do you want to remove that usage in this PR also? https://github.com/edx/edx-platform/blob/2b4a04bbf948f6a41aec1594c7b7991f1694c2b7/lms/djangoapps/courseware/views/views.py#L1126